virttest/qemu_vm.py

```diff
@@ -2425,6 +2425,9 @@ def sort_key(dev):
             options["deflate-on-oom"] = deflate_on_oom
             guest_polling = balloon_params.get("balloon_opt_guest_polling")
             options["guest-stats-polling-interval"] = guest_polling
+            free_report = balloon_params.get("balloon_opt_free_page_reporting")
+            if free_report:
```

Comment:
No need to check this, since :
https://github.com/avocado-framework/avocado-vt/blob/d0c3237d2aa35f9f246a776c6bcec1a069732242/virttest/qemu_devices/qdevices.py#L134-L135

Reply:
Yeah, you are right. Updated. Thanks.